src/pt_br/index.js

```diff
@@ -351,4 +547,8 @@ export const _phoneType = {
 export const _registryType = {
```

Comment:
cammel case sempre @matheusgnreis 

```suggestion
export const _RegistryType = {
```

Reply:
é pascal case, não?

Reply:
é haha
mas pascal case começa com maiúsculo mesmo, camel case que não começa, então eu sugeri certo mas comentei errado :bowtie:

Reply:
Sim, ce sugeriu certo mas eu perguntei porque pensei que você tava usando o `_` como se fosse uma parte da string (?????)

Reply:
O `_` na verdade é porque é um objeto...
Eu quis criar um jeito que diferenciar objetos, sacou?

Reply:
Então sempre pascal case, mas com um `_` no início quando é um objeto pra enumerar status ou afins mapeados da API